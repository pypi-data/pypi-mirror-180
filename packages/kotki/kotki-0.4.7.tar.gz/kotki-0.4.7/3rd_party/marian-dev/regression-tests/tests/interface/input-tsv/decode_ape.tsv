The files are stored in the Adobe / Fireworks CS4 folder in your user-specific Application Data folder ( Windows ) or Application Support folder ( Mac OS ) .	Die Dateien werden in der Adobe / Fireworks CS4 Ordner der benutzerspezifische Anwendungsdaten-Ordner ( Windows ) bzw. " Application Support ( Mac OS ) gespeichert .
Right-click the document in Windows Explorer , choose Properties , and click the PDF tab .	Klicken Sie mit der rechten Maustaste auf das Dokument im Windows-Explorer , wählen Sie " Eigenschaften , " und klicken Sie auf die Registerkarte " PDF " aus .
Also see function literal , and string literal .	Siehe auch function Literal und Stringliteral .
Then perform light touchup in Acrobat Pro or Acrobat Pro Extended after you add the fillable form fields .	Führen Sie dann die helle Retuschierung in Acrobat Pro oder Acrobat Pro Extended , nachdem Sie die ausfüllbaren Formularfelder hinzufügen .
Shift-drag to constrain the movement of the object horizontally , vertically , or diagonally ( in multiples of 45 ° ) .	Halten Sie beim Ziehen des Zeigers über die Bewegung des Objekts horizontal , vertikal oder diagonal einzuschränken ( ein Vielfaches von 45 ° ) .
For web users to see your application , embed it in a web page .	Bei Webbenutzern verwendet wird , um die Anwendung anzuzeigen , die in einer Webseite eingebettet werden .
3D comments added to other views are listed as components of that view in the Model Tree .	3D hinzugefügten Kommentare zu anderen Ansichten als Komponenten anzuzeigen , die in der Modellhierarchie aufgeführt sind .
Display the Property inspector ( Window > Properties ) or panel to use :	Zeigen Sie den Eigenschafteninspektor ( " Fenster " " Eigenschaften " ) oder " verwenden , um :
Its behavior is similar to using the Free Transform command on a rectangle selection .	Das Verhalten ist ähnlich wie bei Verwendung des Befehls " Frei transformieren " auf ein Rechteck .
You can also perform many types of transformations by dragging the bounding box for a selection .	Sie können auch zahlreiche Transformationsarten durchführen , indem Sie den Begrenzungsrahmen für eine Auswahl .
function applyCurrentValues ( ) -applies the values entered through the Symbol Properties panel to the graphic symbol .	Funktion applyCurrentValues ( ) -applies die Werte über das Bedienfeld " Symboleigenschaften " auf das Grafiksymbol eingegeben werden .
Compound shapes can be made up of simple or compound paths , text frames , text outlines , or other shapes .	Zusammengesetzte Formen können einfache oder zusammengesetzte Pfade , Textrahmen , Textgliederung oder andere Formen aufgeholt werden .
Reset to number sets a specific number from which to number list item entries .	Setzen Sie die Nummer , um eine bestimmte Anzahl von der Nummerierung der Listenelemente beginnen soll .
The Gradient tool cannot be used with bitmap or indexed-color images .	Das Verlaufswerkzeug kann nicht mit Bitmap- oder indizierten Farben verwendet werden .
Choose an option from the Key Algorithm menu .	Wählen Sie eine Option aus dem Menü " Algorithm . "
Select Edit > Timeline > Copy Motion .	Wählen Sie " Bearbeiten " " Zeitleiste " > " Kopieren . "
Resizes a frame to fit its content .	Passt die Größe eines Rahmens , um dessen Inhalt einzupassen .
If there are no compilation errors , you will see a Flash Player window showing the Hello World application .	Wenn keine Kompilierungsfehler vorliegen , wird eine Flash Player-Fenster mit der Anwendung " Hello World " angezeigt .
The policy file must load before you can extract data .	Die Richtliniendatei geladen werden müssen , bevor Sie Daten extrahieren können .
For additional shortcuts , see Keys for using the Animation panel in Timeline Mode ( Photoshop Extended ) .	Für weitere Tastenkombinationen finden Sie unter Tastenkombinationen zur Verwendung des Animationsbedienfelds in Zeitleiste Modus ( Photoshop Extended ) .
When you want to see the images in more detail , you can switch the document view to Typical or High Quality ( leaving the preference set to Fast ) .	Wenn Sie die Bilder im Detail anzeigen möchten , können Sie die Dokumentansicht wechseln , um Typisch oder Hohe Qualität ( die Voreinstellung auf Schnell festgelegt ist ) .
Font designers may include optional ligatures that shouldn 't be turned on in all circumstances .	Font Designer können optionale Ligaturen enthalten , die nicht in allen Fällen aktiviert .
Click the New button .	Klicken Sie auf die Schaltfläche " Neu . "
( ) global function you specify the base type of the Vector that the method returns .	( ) -Funktion verwenden Sie den Basistyp der Vektor angeben , dass die Methode zurückgibt .
Alt-click ( Windows ) or Option-click ( Mac OS ) the triangle to open or close a group and the groups nested within it .	Klicken Sie bei gedrückter Alt-Taste ( Windows ) bzw. Wahltaste ( Mac OS ) auf das Dreieck , um eine Gruppe zu öffnen oder zu schließen und die darin enthaltenen verschachtelten Gruppen .
Select the other previews and specify different optimization settings for each preview .	Wählen Sie die anderen Vorschauen und verschiedene Optimierungseinstellungen für jede Vorschau angeben .
Restores the crop margins to zero .	Stellt die Ränder beschneiden , um Null .
Saves the Output panel contents to a text file .	Speichert das Bedienfeld " Ausgabe " Inhalte in eine Textdatei .
To set the sharpness of the glow , drag the Strength slider .	Um die Schärfe des Glühen-Effekts festzulegen , ziehen Sie den Schieberegler " Stärke " aus .
Reduces the tint of the lighter color being trapped ; the darker color remains at 100 % .	Reduziert die Tönung des Einklemmens der helleren Farbe ; die dunklere Farbe bleibt bei 100 % .
This option is intended to be used with an airbrush pen that has a stylus wheel on its barrel and with a graphics tablet that can detect that pen .	Diese Option ist mit einem Airbrush Stift verwendet werden , die mit einem Grafiktablett verfügt über ein Stylus-Rad und Stift erkennen können .
Fireworks exports additional pages for each button in the navigation bar .	Fireworks exportiert zusätzliche Seiten für jede Schaltfläche in der Navigationsleiste .
It 's no longer necessary to check out the story or copy and paste to a different story to view this information .	Es ist nicht mehr notwendig , den Textabschnitt auschecken oder kopieren und in einen anderen Textabschnitt einfügen , um diese Informationen anzuzeigen .
To delete multiple layers , hold Ctrl ( Windows ) or Command ( Mac OS ) while you click the layers you want to delete .	Wenn Sie mehrere Ebenen löschen möchten , halten Sie die Strg-Taste ( Windows ) bzw. die Befehlstaste ( Mac OS ) gedrückt , während Sie auf die Ebenen , die Sie löschen möchten .
For more information , see these videos :	Weitere Informationen finden Sie unter den folgenden Videos :
The Rebuild button appears at the bottom of the Optimize panel when a rebuild is needed .	Die Schaltfläche " Wiederherstellen " wird am unteren Rand des Bedienfelds " Optimieren " aus , wenn Sie eine Neuerstellung erforderlich ist .
Click once to select the baseline of the type you want to change ; click twice to select all of the type .	Klicken Sie einmal , um die Basislinie des Typs auswählen , die Sie ändern möchten , und klicken Sie zweimal , um den gesamten Text auszuwählen .
Select View > Go To , and then select the name of the scene from the submenu .	Wählen Sie " Ansicht " " Gehe zu , und wählen Sie dann den Namen der Szene aus dem Untermenü aus .
This chapter focuses on run-time errors .	Dieses Kapitel befasst sich mit Laufzeitfehlern .
From the Animation panel menu , choose Delete Keyframes .	Wählen Sie im Menü des Animationsbedienfelds die Option " Keyframes löschen . "
Works best with large font sizes and can be slow with large amounts of text .	Dies eignet sich optimal für große Schriftgrößen und kann für große Textmengen verlangsamt werden .
Choose File > Automate > Conditional Mode Change .	Wählen Sie " Datei " " Automatisch " > " Bedingter Modus " Ändern . "
After you optimize your graphics , the next step is to export them for use on the web .	Nachdem Sie die Grafiken optimieren , ist der nächste Schritt , um sie für den Einsatz im Internet exportieren .
Glyph Scaling values can range from 50 % to 200 % .	Glyphenabstand " Werte können in einem Bereich von 50 % bis 200 % .
The following table lists the keyboard shortcuts for performing Insert menu actions in Contribute .	In der folgenden Tabelle sind die Tastenkombinationen zum Ausführen im Menü " Einfügen " in Contribute .
Each of those bits of information is stored somewhere in the computer 's memory .	Jeder dieser Informationen irgendwo im Speicher des Computers gespeichert wird .
Ctrl ‑ click ( Windows ) or Command-click ( Mac OS ) the path thumbnail in the Paths panel .	Klicken Sie bei gedrückter Strg- ( Windows ) bzw. Befehlstaste ( Mac OS ) im Pfadebedienfeld den Pfad der Miniaturansicht .
For more information on using this reference point , see Position a custom-positioned anchored object .	Weitere Informationen zur Verwendung dieses Referenzpunkts finden Sie unter Positionieren von benutzerspezifisch positionierten verankerten Objekts .
For a video on using keyboard shortcuts , see http : / / www.adobe.com / go / vid0066 www.adobe.com / go / vid0066 .	Ein Video zur Verwendung von Tastaturbefehlen finden Sie unter http : / / www.adobe.com / go / vid0066 _ de www.adobe.com / go / vid0066 .
Different parameters are available for slide and form screens .	Verschiedene Parameter für Folien- und Formularbildschirme verfügbar sind .
Choose Window > Channels to display the Channels panel .	Wählen Sie " Fenster " " Kanäle , um im Kanälebedienfeld angezeigt werden .
For example , suppose you want to update the content of a formatting table in a monthly magazine .	Angenommen , Sie möchten den Inhalt einer Formatierung der Tabelle in eine monatlich erscheinende Zeitschrift zu aktualisieren .
For more information on using relative and full URLs , see an HTML reference .	Weitere Informationen zur Verwendung von relativen und vollständigen URLs finden Sie unter einem HTML-Handbuch .
Choose Select > Feather .	Wählen Sie " Auswahl " " Weiche Kante . "
To copy the selection and offset the duplicate by 10 pixels , press Alt + Shift ( Windows ) or Option + Shift ( Mac OS ) , and press an arrow key .	Wenn Sie die Auswahl kopieren und das Duplikat um 10 Pixel versetzen , drücken Sie die Alt- + Umschalt- ( Windows ) bzw. Wahl- + Umschalttaste ( Mac OS ) gedrückt und drücken Sie eine Pfeiltaste .
When unchecked , it hides any polygons touched by the selection .	Wenn diese Option deaktiviert ist , werden alle Polygone berührt , die durch die Auswahl ausgeblendet wird .
Illustrator supports multiple scripting environments ( including Microsoft Visual Basic , AppleScript , JavaScript , and ExtendScript ) .	Illustrator unterstützt mehrere Skriptumgebungen ( einschließlich Microsoft Visual Basic AppleScript , JavaScript- und " ExtendScript " ) .
Select Insert > Enclosure > From the Internet .	Wählen Sie " Einfügen " " Anlage " > " Aus dem Internet herstellen .
Choose File > Save As , and choose Photoshop DCS 1.0 or Photoshop DCS 2.0 from the Format menu .	Wählen Sie " Datei " " Speichern unter , " und wählen Sie " Photoshop DCS 1.0 oder Photoshop DCS 2.0 aus dem Menü " Format " aus .
Choose Edit > Preferences ( Windows ) or Bridge CS4 > Preferences ( Mac OS ) , and click Thumbnails .	Wählen Sie " Bearbeiten " " Voreinstellungen " ( Windows ) bzw. " Bridge CS4 " > " Voreinstellungen " ( Mac OS ) und klicken Sie auf " Miniaturen . "
Locate and select a replacement file .	Suchen und wählen Sie eine neue Datei aus .
Drag the Scale slider or enter a value to specify the size of the pattern .	Ziehen Sie den Schieberegler , oder geben Sie einen Wert ein , um die Größe des Musters anzugeben .
( Optional ) Set any of the following options .	( Optional ) Legen Sie eine der folgenden Optionen aus .
As long as any reference to the object exists , the garbage collection system will not recover the memory that the object occupies .	Solange jeder Verweis auf das Objekt vorhanden ist , wird die Speicherbereinigung des Systems nicht auf den Arbeitsspeicher wiederherstellen , der das Objekt einnimmt .
The numbered list you define determines whether paragraph numbering maintains sequential numbering across documents in a book .	Die Liste , die Sie definieren , legt fest , ob die Absatznummerierung behält fortlaufende Nummerierung über mehrere Dokumente in einem Buch nummeriert .
The Adjust Color filter allows you to finely control the color attributes of the selected object , including contrast , brightness , saturation , and hue .	Der Filter " die Option " Farbe anpassen " können Sie die Attribute des ausgewählten Objekts , einschließlich Kontrast , Helligkeit , Sättigung und Farbton fein steuern .
Choose this option for mismatches that you want to know about and may need to correct before final output .	Wählen Sie diese Option für Abweichungen aus , die Sie erfahren möchten und müssen vor der Endausgabe korrigieren .
If you are changing the properties of multiple fields , you can leave the Properties dialog box open .	Wenn Sie die Eigenschaften mehrerer Felder ändern , können Sie das Dialogfeld " Eigenschaften " geöffnet lassen .
If you specify multiple pages , hold down Alt ( Windows ) or Option ( Mac OS ) while placing the file to place them all at the same time , overlapping each other .	Wenn Sie mehrere Seiten angeben möchten , halten Sie die Alt-Taste ( Windows ) bzw. die Wahltaste ( Mac OS ) gedrückt , während Sie die Datei platzieren , um sie alle gleichzeitig zu platzieren , gegenseitig überlappen .
To change the reference point , click a white square on the reference point locator before you enter the values .	Wenn Sie den Referenzpunkt ändern möchten , klicken Sie auf ein weißes Quadrat im Ursprungssymbol , bevor Sie die Werte eingeben .
Collapses the branches in the navigation tree that do not contain the selected element .	Blendet die Zweige im Navigationsbaum , die nicht mit dem ausgewählten Element .
Hold down Alt ( Windows ) or Option ( Mac OS ) , position the pointer over a swatch ( the pointer turns into scissors ) , and click .	Halten Sie die Alt-Taste ( Windows ) bzw. die Wahltaste ( Mac OS ) gedrückt , positionieren Sie den Zeiger über einem Farbfeld ( der Zeiger wird in der Schere ) , und klicken Sie auf .
To preserve transparency , use the File > Import To Stage or Import To Library command for importing .	Um die Transparenz beizubehalten , verwenden Sie den Befehl " Datei " " In Bühne importieren " oder " In Bibliothek importieren " für den Import .
Set the zoom control to 400 % , or higher if you like , and edit the symbol to change its look .	Legen Sie die Zoomsteuerung bis 400 % oder höher , wenn Sie möchten , und bearbeiten Sie das Symbol , um deren Aussehen zu ändern .
Uses the area of the component to clip the underlying geometry as a mask would .	Verwendet die Fläche der Komponente auf die zugrunde liegende Geometrie als Maske geclippt wird .
onLoadComplete ( ) : In the onLoadComplete ( ) method the code creates the Shader object named shader .	onLoadComplete ( ) : Klicken Sie im onLoadComplete ( ) -Methode der Code des Shader-Objekts mit dem Namen Shader erstellt .
See Using HTML-formatted text in http : / / www.adobe.com / go / learn _ fl _ cs4 _ learningAS2 _ en Learning ActionScript 2.0 in Adobe Flash .	Siehe Verwenden von HTML-formatierten Text in http : / / www.adobe.com / go / learn _ fl _ cs4 _ learningAS2 _ de ActionScript 2.0 in Adobe Flash - Arbeitshandbuch .
Specifies whether the distance measurement caption is Inside or on Top of the measurement line .	Gibt an , ob die Abstandmessung Bildunterschrift Innen oder über die Messlinie ist .
If it 's important for other objects to be notified of the exception , your code must re-throw the exception to start the notification process again .	Wenn es wichtig ist , dass andere Objekte der Ausnahme benachrichtigt werden soll , muss Ihr Code die Ausnahme erneut auszulösen , um den Benachrichtigungsprozess erneut starten .
Choose Object > Interactive > Convert To Button .	Wählen Sie " Objekt " " Interaktiv " > " In Schaltfläche konvertieren .
The icon next to an instance of the search results indicates the search area in which the instance appears .	Das Symbol neben einer Instanz der Suchergebnisse gibt die Suche in der Instanz angezeigt .
This is true whether the subclass is in the same package or in a different package .	Dies gilt unabhängig davon , ob die Unterklasse im gleichen Paket oder in einem anderen Paket ist .
For more information about animating with ActionScript 3.0 , see the fl.motion classes in the ActionScript 3.0 Language and Components Reference .	Weitere Informationen zum Animieren mit ActionScript 3.0 finden Sie in der fl.motion Klassen in ActionScript 3.0-Komponenten-Referenzhandbuch nachzuschlagen .
Enter a value of 90 ° , and click Copy to create a copy flush left of the first tile .	Geben Sie einen Wert von 90 ° ein , und klicken Sie auf " Kopieren , " um eine Kopie bündig links von der ersten Kachel erstellen .
Paint with a value of gray to add the channel 's color at a lower intensity .	Malen Sie mit einem Wert von Grau hinzufügen , um die Farbe des Kanals auf einer niedrigeren Intensität .
It is measured as a location on the standard color wheel , expressed as a degree between 0 ° and 360 ° .	Es wird als eine Stelle auf dem standardmäßigen Farbkreis als Grad zwischen 0 ° und 360 ° gemessen .
When the action is played back , the work path is set to the recorded path .	Wenn Sie die Aktion wiedergegeben wird , ist der Arbeitspfad zum Festlegen des Pfads aufgezeichnet .
automatically locks tags specified in a loaded DTD file .	Sperren automatisch Tags in einer geladenen DTD-Datei festgelegt haben .
In the graph , each bar represents an individual frame of the document .	Klicken Sie im Diagramm jedes Balkens stellt einen einzelnen Frame des Dokuments .
If you want to manually remove outdated drafts , press Control + Shift while selecting View > Refresh Drafts .	Wenn Sie veraltete Entwürfe manuell zu entfernen , drücken Sie Strg + Umschalt gedrückt und wählen Sie " Ansicht " " Entwürfe aktualisieren . "
See Example : Creating scrolling text in Learning ActionScript 2.0 in Adobe Flash .	Weitere Informationen finden Sie unter Beispiel : Erstellen von Lauftext in ActionScript 2.0 in Adobe Flash - Arbeitshandbuch .
This allows you to modify the XML packet before attempting to revalidate it .	Auf diese Weise können Sie das XML-Paket ändern , bevor Sie sie erneut validieren .
For more information about options in this dialog box , see Setting the Enable Publishing Server dialog box options .	Weitere Informationen zu den Optionen in diesem Dialogfeld finden Sie unter Festlegen der Publishing Server aktivieren " im Dialogfeld " Optionen . "
For more information , see Completing the Publish New Linked Blog Entries dialog box options .	Weitere Informationen finden Sie unter Ausfüllen des Dialogfelds " Neue verknüpfte Blogeinträge veröffentlichen die Optionen .
In preparing to convert web pages to PDF , consider the following factors , which affect how you approach the conversion process :	Klicken Sie zum Vorbereiten von Webseiten in das PDF-Format konvertieren , sollten Sie die folgenden Faktoren , die sich darauf auswirken , wie Sie den Konvertierungsprozess Ansatz :
Choose Adobe PDF > Batch Conversion .	Wählen Sie " Adobe PDF " " Konvertierungsbereich " Stapelverarbeitung " aus .
Vector graphics describe images by using lines and curves , called vectors , that also include color and position properties .	Vektorgrafiken beschreiben Bilder mithilfe von Linien und Kurven , so genannten Vektoren , die auch die Farbe und die Position von Eigenschaften enthalten .
Choose Window > Info or click the Info panel tab to view the panel .	Wählen Sie " Fenster " " Info " oder klicken Sie im Bedienfeld " Info " klicken , um das Bedienfeld anzuzeigen .
Use the Edit PDF Portfolio pane to customize the layout , welcome page , header , and color scheme .	Verwenden Sie im Fenster " Portfolio bearbeiten " auf das Layout , die Willkommensseite , Kopf- und Farbschema anpassen .
Alternatively , enter a value in the Luminance box .	Sie können auch einen Wert in das Feld Intensität eingeben .
