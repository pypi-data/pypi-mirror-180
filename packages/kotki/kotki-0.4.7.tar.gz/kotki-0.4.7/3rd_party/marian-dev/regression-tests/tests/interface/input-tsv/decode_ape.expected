Die Dateien werden im Adobe / Fireworks CS4 -Ordner in Ihrem benutzerspezifischen Anwendungsdaten-Ordner ( Windows ) bzw. im Application Support-Ordner ( Mac OS ) gespeichert .
Klicken Sie mit der rechten Maustaste auf das Dokument im Windows-Explorer , wählen Sie " Eigenschaften , " und klicken Sie auf die Registerkarte " PDF . "
Siehe auch function Literal und string Literal .
Führen Sie dann die helle Retuschierung in Acrobat Pro oder Acrobat Pro Extended durch , nachdem Sie die ausfüllbaren Formularfelder hinzugefügt haben .
Halten Sie beim Ziehen die Umschalttaste gedrückt , um die Bewegung des Objekts horizontal , vertikal oder diagonal einzuschränken ( ein Vielfaches von 45 Prozent ) .
Bei Webbenutzern , um die Anwendung anzuzeigen , die in einer Webseite eingebettet ist .
3D-Kommentare , die zu anderen Ansichten hinzugefügt werden , werden als Komponenten in der Modellhierarchie aufgeführt .
Zeigen Sie den Eigenschafteninspektor ( " Fenster " > " Eigenschaften " ) oder das Bedienfeld an :
Das Verhalten ist ähnlich wie bei Verwendung des Befehls " Frei transformieren " über eine Rechteckauswahl .
Sie können auch zahlreiche Transformationsarten durchführen , indem Sie den Begrenzungsrahmen für eine Auswahl ziehen .
Funktion applyCurrentValues ( ) -applies die Werte , die über das Bedienfeld " Symboleigenschaften " auf das Grafiksymbol eingegeben werden .
Zusammengesetzte Formen können einfache oder zusammengesetzte Pfade , Textrahmen , Textkonturen oder andere Formen sein .
Setzen Sie die Nummer auf eine bestimmte Anzahl , ab der die Nummerierung der Listenelemente beginnen soll .
Das Verlaufswerkzeug kann nicht mit Bitmap- oder indizierten Farben verwendet werden .
Wählen Sie eine Option aus dem Menü " Key Algorithm . "
Wählen Sie " Bearbeiten " > " Zeitleiste " > " Bewegung kopieren . "
Passt die Größe eines Rahmens an dessen Inhalt an .
Wenn keine Kompilierungsfehler vorliegen , wird ein Flash Player-Fenster mit der Anwendung " Hello World " angezeigt .
Die Richtliniendatei muss geladen werden , bevor Sie Daten extrahieren können .
Weitere Tastenkombinationen finden Sie unter " Tastenkombinationen " zur Verwendung des Animationsbedienfelds im Zeitleistenmodus ( Photoshop Extended ) .
Wenn Sie die Bilder im Detail anzeigen möchten , können Sie die Dokumentansicht wechseln , um Typisch oder Hohe Qualität ( die Voreinstellung auf Schnell festgelegt ist ) zu wählen .
Font Designer können optionale Ligaturen enthalten , die nicht in allen Fällen aktiviert werden sollten .
Klicken Sie auf die Schaltfläche " Neu . "
Funktion global ( ) Sie geben den Basistyp des Vektors an , den die Methode zurückgibt .
Klicken Sie bei gedrückter Alt-Taste ( Windows ) bzw. Wahltaste ( Mac OS ) auf das Dreieck , um eine Gruppe zu öffnen oder zu schließen und die darin enthaltenen verschachtelten Gruppen zu schließen .
Wählen Sie die anderen Vorschauen aus , und legen Sie verschiedene Optimierungseinstellungen für jede Vorschau fest .
Stellt die Ränder beschneiden auf Null zurück .
Speichert die Inhalte des Bedienfelds " Ausgabe " in eine Textdatei .
Um die Schärfe des Glühen-Effekts festzulegen , ziehen Sie den Schieberegler " Stärke . "
Reduziert die Tönung des Einklemmens der helleren Farbe ; die dunklere Farbe bleibt bei 100 Prozent .
Diese Option soll mit einem Airbrush-Stift verwendet werden , der über ein Stylus-Rad verfügt und mit einem Grafiktablett diesen Stift erkennen kann .
Fireworks exportiert zusätzliche Seiten für jede Schaltfläche in der Navigationsleiste .
Es ist nicht mehr notwendig , den Textabschnitt auszuchecken oder zu kopieren und in einen anderen Textabschnitt einzufügen , um diese Informationen anzuzeigen .
Wenn Sie mehrere Ebenen löschen möchten , halten Sie die Strg-Taste ( Windows ) bzw. die Befehlstaste ( Mac OS ) gedrückt , während Sie auf die Ebenen klicken , die Sie löschen möchten .
Weitere Informationen finden Sie unter den folgenden Videos :
Die Schaltfläche " Wiederherstellen " wird am unteren Rand des Bedienfelds " Optimieren " angezeigt , wenn eine Neuerstellung erforderlich ist .
Klicken Sie einmal , um die Basislinie des Typs auszuwählen , den Sie ändern möchten , und klicken Sie zweimal , um den gesamten Text auszuwählen .
Wählen Sie " Ansicht " > " Gehe zu , " und wählen Sie dann den Namen der Szene aus dem Untermenü aus .
Dieses Kapitel befasst sich mit Laufzeitfehlern .
Wählen Sie im Menü des Animationsbedienfelds die Option " Keyframes löschen . "
Dies eignet sich optimal für große Schriftgrößen und kann bei großen Textmengen verlangsamt werden .
Wählen Sie " Datei " > " Automatisch " > " Bedingter Modus " Ändern . "
Nachdem Sie die Grafiken optimiert haben , ist der nächste Schritt , sie für den Einsatz im Internet zu exportieren .
Glyphenabstandswerte können in einem Bereich von 50 Mail bis 200 Prozent liegen .
In der folgenden Tabelle sind die Tastenkombinationen zum Ausführen von Einfügen-Menüaktionen in Contribute aufgeführt .
Jede dieser Informationen wird irgendwo im Speicher des Computers gespeichert .
Klicken Sie bei gedrückter Strg- ( Windows ) bzw. Befehlstaste ( Mac OS ) auf die Pfadminiatur im Pfadebedienfeld .
Weitere Informationen zur Verwendung dieses Referenzpunkts finden Sie unter " Positionieren von benutzerspezifisch positionierten verankerten Objekts . "
Ein Video zur Verwendung von Tastaturbefehlen finden Sie unter http : / / www.adobe.com / go / vid0066 www.adobe.com / go / vid0066 .
Verschiedene Parameter sind für Folien- und Formularbildschirme verfügbar .
Wählen Sie " Fenster " > " Kanäle , " um das Kanälebedienfeld anzuzeigen .
Angenommen , Sie möchten den Inhalt einer Formatierungstabelle in einer monatlich erscheinenden Zeitschrift aktualisieren .
Weitere Informationen zur Verwendung von relativen und vollständigen URLs finden Sie unter einem HTML-Handbuch .
Wählen Sie " Auswahl " > " Weiche Kante . "
Wenn Sie die Auswahl kopieren und das Duplikat um 10 Pixel versetzen möchten , drücken Sie die Alt- + Umschalt- ( Windows ) bzw. Wahl- + Umschalttaste ( Mac OS ) , und drücken Sie eine Pfeiltaste .
Wenn diese Option deaktiviert ist , werden alle Polygone berührt , die durch die Auswahl berührt wurden .
Illustrator unterstützt mehrere Skriptumgebungen ( einschließlich Microsoft Visual Basic , AppleScript , JavaScript und ExtendScript ) .
Wählen Sie " Einfügen " > " Anlage " > " Aus dem Internet herstellen . "
Wählen Sie " Datei " > " Speichern unter , " und wählen Sie " Photoshop DCS 1.0 " oder " Photoshop DCS 2.0 " aus dem Menü " Format " aus .
Wählen Sie " Bearbeiten " > " Voreinstellungen " ( Windows ) bzw. " Bridge CS4 " > " Voreinstellungen " ( Mac OS ) , und klicken Sie auf " Miniaturen . "
Suchen und wählen Sie eine neue Datei aus .
Ziehen Sie den Schieberegler " skalieren , " oder geben Sie einen Wert ein , um die Größe des Musters anzugeben .
( Optional ) Legen Sie eine der folgenden Optionen fest .
Solange jeder Verweis auf das Objekt vorhanden ist , wird die Speicherbereinigung des Systems den Arbeitsspeicher nicht wiederherstellen , der das Objekt einnimmt .
Die Nummerierungsliste , die Sie definieren , legt fest , ob die Absatznummerierung beibehält , und zwar fortlaufende Nummerierung über mehrere Dokumente in einem Buch .
Mit dem Filter " Farbe anpassen " können Sie die Attribute des ausgewählten Objekts fein steuern , einschließlich Kontrast , Helligkeit , Sättigung und Farbton .
Wählen Sie diese Option für Abweichungen aus , die Sie erfahren möchten und müssen diese vor der Endausgabe korrigieren .
Wenn Sie die Eigenschaften mehrerer Felder ändern , können Sie das Dialogfeld " Eigenschaften " geöffnet lassen .
Wenn Sie mehrere Seiten angeben möchten , halten Sie die Alt-Taste ( Windows ) bzw. die Wahltaste ( Mac OS ) gedrückt , während Sie die Datei platzieren , damit sie alle gleichzeitig ineinander verschachtelt wird .
Wenn Sie den Referenzpunkt ändern möchten , klicken Sie auf ein weißes Quadrat im Ursprungssymbol , bevor Sie die Werte eingeben .
Blendet die Zweige im Navigationsbaum , die nicht das ausgewählte Element enthalten , aus .
Halten Sie die Alt-Taste ( Windows ) bzw. die Wahltaste ( Mac OS ) gedrückt , positionieren Sie den Zeiger über einem Farbfeld ( der Zeiger wird in der Schere dargestellt ) , und klicken Sie .
Um die Transparenz beizubehalten , verwenden Sie den Befehl " Datei " > " In Bühne importieren " oder " In Bibliothek importieren " für den Import .
Legen Sie die Zoomsteuerung auf 400 Prozent oder höher fest , wenn Sie möchten , und bearbeiten Sie das Symbol , um deren Aussehen zu ändern .
Verwendet die Fläche der Komponente , um die zugrunde liegende Geometrie auszuklammern , die als Maske geclippt wird .
onLoadComplete ( ) : In der onLoadComplete ( ) -Methode wird der Code des Shader-Objekts mit dem Namen Shader erstellt .
Siehe Verwenden von HTML-formatiertem Text in http : / / www.adobe.com / go / learn _ fl _ cs4 _ learningAS2 _ de " ActionScript 2.0 in Adobe Flash - Arbeitshandbuch . "
Gibt an , ob die Abstandsmessung über die Beschriftung Innen oder über die Messlinie erfolgt .
Wenn es wichtig ist , dass andere Objekte der Ausnahme benachrichtigt werden sollen , muss Ihr Code die Ausnahme erneut auslösen , um den Benachrichtigungsprozess erneut starten zu können .
Wählen Sie " Objekt " > " Interaktiv " > " In Schaltfläche konvertieren . "
Das Symbol neben einer Instanz der Suchergebnisse gibt den Suchbereich an , in dem die Instanz angezeigt wird .
Dies gilt unabhängig davon , ob die Unterklasse im gleichen Paket oder in einem anderen Paket ist .
Weitere Informationen zum Animieren mit ActionScript 3.0 finden Sie unter den fl.motion-Klassen im Komponenten-Referenzhandbuch zu ActionScript 3.0 .
Geben Sie einen Wert von 90 Prozent ein , und klicken Sie auf " Kopieren , " um eine Kopie bündig links von der ersten Kachel zu erstellen .
Malen Sie mit einem Wert von Grau , um die Farbe des Kanals einer niedrigeren Intensität hinzuzufügen .
Es wird als eine Stelle auf dem standardmäßigen Farbkreis als Grad zwischen 0 und 360 Prozent gemessen .
Wenn die Aktion wiedergegeben wird , wird der Arbeitspfad zum Festlegen des aufgezeichneten Pfads festgelegt .
Sperren automatisch Tags in einer geladenen DTD-Datei .
In der Grafik repräsentiert jeder Balken einen einzelnen Frame des Dokuments .
Wenn Sie veraltete Entwürfe manuell entfernen möchten , drücken Sie Strg + Umschalt und wählen Sie " Ansicht " > " Entwürfe aktualisieren . "
Weitere Informationen finden Sie unter Beispiel : Erstellen von Lauftext in Learning ActionScript 2.0 in Adobe Flash .
Auf diese Weise können Sie das XML-Paket ändern , bevor Sie es erneut validieren .
Weitere Informationen zu den Optionen in diesem Dialogfeld finden Sie unter " Festlegen der Optionen für Publishing Server aktivieren . "
Weitere Informationen finden Sie unter Ausfüllen des Dialogfelds " Neue verknüpfte Blogeinträge veröffentlichen " in den Optionen .
Bei der Erstellung zur Konvertierung von Webseiten in das PDF-Format sollten Sie die folgenden Faktoren berücksichtigen , die sich darauf auswirken , wie Sie den Konvertierungsprozess vorgehen :
Wählen Sie " Adobe PDF " > " Stapelverarbeitung " aus .
Vektorgrafiken beschreiben Bilder mithilfe von Linien und Kurven , so genannten Vektoren , die auch die Farbe und die Position von Eigenschaften enthalten .
Wählen Sie " Fenster " > " Info , " oder klicken Sie im Bedienfeld " Info , " um das Bedienfeld anzuzeigen .
Verwenden Sie das Fenster " Portfolio bearbeiten , " um das Layout , die Willkommensseite , das Kopf- und Farbschema anzupassen .
Sie können auch einen Wert in das Feld Intensität eingeben .
